Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -123,6 +123,16 @@ async function run() {
             tl.setVariable(webAppUri, publishingProfile.destinationAppUrl);
         }
 
+        if(publishingProfile && publishingProfile.destinationAppUrl) {
+            try{
+                await azureRESTUtility.warmupAzureAppService(publishingProfile.destinationAppUrl);
+                console.log(tl.loc("appServiceWarmupedUpSuccessfully"));
+            } catch (error) {
+                console.log(tl.loc('failedToWarmupAppService'));
```

Comment:
Do we need this in console. Warm is just as best case to increase the success rate. Isn't it enough to be in debug logs alone.